swap_meet/item.py

```diff
@@ -1,2 +1,19 @@
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, category="", condition=0, age=None):
+        self.category = category
+        self.condition = condition
+        self.age = age
+    
+    def __str__(self):
+        return "Hello World!"
+    
+    def condition_description(self):
+        description = {
+            0: "new", 
+            1: "barely used",
+            2: "good for a couple of days",
+            3: "found it in Paris",
+            4: "light blue color",
+            5: "has weird smell"
+            }
+        return description[self.condition]
```

Comment:
Condition can be a floating point value, but indexing into an list will throw an error with a float (TypeError: list indices must be integers or slices, not float).  Casting the condition to an int will result in dropping the decimal and avoid the error:
```suggestion
        return description[int(self.condition)]
```